tools_considered_and_the_screening_decisions.md

```diff
@@ -0,0 +1,178 @@
+|Syras                                                                          |         0|         1|         0|no RITL-based AL cycle                                                      |
+|SyRF: Systematic Review Facility                                               |         1|         1|         0|no RITL-based AL cycle                                                      |
+|SysRev                                                                         |         1|         1|         0|no RITL-based AL cycle                                                      |
+|Systematic Review Accelerator / SR-Accelerator / SRA                           |         1|         1|         0|no RITL-based AL cycle                                                      |
+|TaskExchange                                                                   |         1|         1|         0|no RITL-based AL cycle                                                      |
+|TextBlob: Simplified Text Processing                                           |         0|         1|         0|no software                                                                 |
+|Thoth                                                                          |         1|         1|         0|no RITL-based AL cycle                                                      |
+|Trialstreamer                                                                  |         1|         0|         0|limited to COVID19                                                          |
+                           
+
+
+
+## Search 1
+
+We consider screening tools that were part of a comparison in papers that cite the 2020 paper by Harrison et al. and/or the 2021 paper by Van de Schoot et al. as of December 2022.
+
+On 6/12/2022, the Scopus database contained 40 papers that cited the Harrison 2020 paper. Of these results, 22 could be downloaded directly from the Scopus website. Of the remaining 18, 17 PDFs could be retrieved from different websites and of only 1 article no PDF version was available (Bravo 2021), so the htm-link was saved instead. 
+
+There is one paper that appears twice in this set of 40, namely Foulquier 2021, of which a Foulquier 2022 is a French translation. Both versions were retrieved and considered for the sake of completeness. 
+
+The same procedure was repeated to retrieve papers that cite Van de Schoot et al. 2020. 
+
+This set of papers was then screened and classified as relevant or irrelevant. A paper was considered relevant if the authors perform a comparison of multiple different systematic reviewing softwares. A paper was considered irrelevant if it lacked such a comparison, for example if its objective was to illustrate an application of such software, or if they assess or introduce one systematic reviewing tool, but do not make a comparison to other tools. Keeping these criteria and after removing duplicates, 13 papers were kept for consideration.
```

Comment:
Why are papers that assess or introduce one systematic reviewing tool excluded? This could still be similar active learning screening tools right?

Reply:
Because such papers have already been found and described by the overview papers we identified. You could consider our project a review of review papers